rs/crypto/internal/crypto_lib/threshold_sig/bls12_381/src/crypto.rs

```diff
@@ -239,6 +239,7 @@ pub(crate) fn verify_individual_sig(
             algorithm: AlgorithmId::ThresBls12_381,
             public_key_bytes: PublicKeyBytes::from(public_key).0.to_vec(),
             sig_bytes: IndividualSignatureBytes::from(signature).0.to_vec(),
+            msg_hash: Some(message.to_vec()),
```

Comment:
Similar comment as for multi-sigs. `message` here is for many types not a hash. As far as I can see (based on `pub trait Crypto`), it is only a hash for `CertificationContent`, but not for `CatchUpContent`, `CatchUpContentProtobufBytes`, `RandomBeaconContent`, and `RandomTapeContent`.

Same for `verify_combined_sig`.